src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -96,6 +96,12 @@ class WorkspaceMembersPage extends React.Component {
             this.validate();
         }
 
+        if (prevProps.policyMemberList !== this.props.policyMemberList) {
+            this.setState((prevState) => ({
+                selectedEmployees: _.intersection(prevState.selectedEmployees, _.keys(this.props.policyMemberList)),
+            }));
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/28551

This is an excellent start to aligning the `policyMembers`. The attached issue is another case in which we want to keep the selected optimistic members' `accountID` aligned with the new `accountID` from the API response, so the selection remains.